_data/languages.yml

```diff
@@ -1,30 +1,1065 @@
---- 
+---
+-
+  code: af
+  name: Afrikaans
+  family: ine
+  scripts: [ Latn ]
+  typology:
+    - SVO, VSO
+    - analytic
+  variants: [ za ]
+
+-
+  code: ar
+  name: Arabic
+  family: sem
+  scripts: [ Arab, Syrc ]
+  typology:
+    - VSO
+    - fusional
+  variants: [ dz, bh, eg, iq, jo, kw, lb, ly, ma, om, qa, sa, sy, tn, ae, ye, sa ]
```

Comment:
`sa` is listed twice.

Palestine (`ps`) is missing.

Also Mauritania and Sudan.  And it's also co-official in Somalia, Eritrea and Djibouti.

It's also been spoken continuously in what's now Turkey like a half a millennium longer than Turkish has.